src/private_key.rs

```diff
@@ -173,8 +174,45 @@ impl<'de> Deserialize<'de> for PrivateKey {
             .and_then(move |bytes| PrivateKey::from_slice(&bytes))
             .map_err(serde::de::Error::custom)
 
-        // Ok(PrivateKey::from_slice(&bytes)?)
-        //
+        //Ok(PrivateKey::from_slice(&bytes)?)
+    }
+}
+
+impl fmt::LowerHex for PrivateKey {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if f.alternate() {
+            let res = write!(f, "0x");
```

Comment:
TBH it could be shortened and simplified with just `write!(f, "0x")?`

Reply:
nope, the 0x is only written when you pass the alternate modifier.

Reply:
Yeah I just meant the branches with `is_err()`. It could be simplified as just `write!(...)?`